Subject: Enable use of pcre even if cross compiling
From: Marc Kleine-Budde <mkl@penutronix.de>

lighttpd thinks that pcre-config cannot be used when cross-compiling,
but lfu fixed pcre-config up, so that we can use it.

This patch enables pcre in lighttpd.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: lighttpd-1.4.20/configure.in
===================================================================
--- lighttpd-1.4.20.orig/configure.in
+++ lighttpd-1.4.20/configure.in
@@ -252,7 +252,7 @@ AC_ARG_WITH(pcre, AC_HELP_STRING([--with
     [WITH_PCRE=$withval],[WITH_PCRE=yes])
 AC_MSG_RESULT([$WITH_PCRE])
 
-if test "x$cross_compiling" = xno -a "$WITH_PCRE" != "no"; then
+if test "$WITH_PCRE" != "no"; then
   AC_PATH_PROG(PCRECONFIG, pcre-config)
 
   if test x"$PCRECONFIG" != x; then
